drivers/SmartThings/zwave-sensor/src/everspring-motion-light-sensor/init.lua

```diff
@@ -32,11 +33,23 @@ local function device_added(driver, device)
   device:send(SensorBinary:Get({ sensor_type = SensorBinary.sensor_type.MOTION }))
 end
 
+local wakeup_notification = nil
+local version = require "version"
+if version.api == 6 then
+  --TODO remove once this happens properly for subdrivers that dont override the default
```

Comment:
There are more subdrivers that rely on the battery default for WakeUpNotifications to issue `device:refresh()`. That commit is incoming and will affect several more sub drivers with a similar change as is here.

Reply:
They were not fixed in the initial commit because there weren't any test that verify the refresh for several of these devices.